static/js/portico/help.js

```diff
@@ -10,6 +10,10 @@ function registerCodeSection($codeSection) {
     const $li = $codeSection.find("ul.nav li");
     const $blocks = $codeSection.find(".blocks div");
 
+    // Activating skip-link
+    const $skiplink = $codeSection.find(".skip-link");
+    $skiplink.addClass("active");
+
```

Comment:
The comment here seems unnecessary, since it's clear from reading the code what it does (which is a nice thing!)